### PR TITLE
Add simple footer and make the app flexbox

### LINKED ISSUE #98: Footer
The upcoming dashboard needs a footer.

Take a look at the one from the existing page.

### PR DESCRIPTION
**Pull request by OtterBoops**
Bad commit management, but I need it so `<main>` can take up the remaining space, excluding header and footer.

Also fixes #98

### CODE REVIEW

File: dashboard/src/components/Footer.vue [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      Coded with lots of <img src="https://discord.com/assets/25c09e6fde32411da2b0da00f5cb9c84.svg" alt="coffee_logo"> by 
+      <a href="https://npstr.space/">Napster</a>
+    </p>
+  </footer>
+</template>
+
+<style lang="scss">
+  footer {
+
+    font-size: 13px;
```
Comment (napstr) [inline dashboard/src/components/Footer.vue:13]: Bulma comes with a bunch of css classes for this. See https://github.com/wolfiabot/Wolfia/blob/master/dashboard/src/components/LogIn.vue#L20 for example

File: dashboard/src/App.vue [hunk 2]
```diff
@@ -90,6 +64,26 @@ export default {
...
+.Header, .Footer {
+	width: 100%;
+}
+
+main {
+	flex: auto;
+}
+
+.Footer {
+	margin-bottom: 5px;
```
Comment (napstr) [inline dashboard/src/App.vue:86]: The other parts of the web frontend codebase is currently sticking to `em` units. If there a reason in this specific case to use px, let's document it.

File: dashboard/src/components/Footer.vue [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+    img {
+      height: 13px;
```
Comment (napstr) [inline dashboard/src/components/Footer.vue:16]: The other parts of the web frontend codebase is currently sticking to `em` units. If there a reason in this specific case to use px, let's document it.

File: dashboard/src/App.vue [hunk 4]
```diff
@@ -90,6 +64,26 @@ export default {
 	font-family: "Avenir", Helvetica, Arial, sans-serif;
 	-webkit-font-smoothing: antialiased;
 	-moz-osx-font-smoothing: grayscale;
+	box-sizing: border-box;
+	display: flex;
```
Comment (napstr) [inline dashboard/src/App.vue:68]: Not sure if this manual management of flexbox stuff is necessary. Have a look at the bulma docs, for example [columns](https://bulma.io/documentation/columns/basics/).

Comment (OtterBoops) [reply dashboard/src/App.vue]: Containers are block by default and we don't really want `#app` to be column in its entirety.

**Comment by napstr** [comment]
LGTM

**Pull request merged**